src/components/recipes/create/RecipesCreate.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { useState } from 'react';
+import { FormEvent, FC } from 'react';
+import { useCreateRecipe } from 'hooks/recipes';
+import { RequestParamsTypeCreateRecipe } from 'types/api';
+import StringsHelper from 'helpers/StringsHelper';
+import { Button, Card, Container, Input } from 'styled/UIComponents';
+import { useNavigate } from 'react-router-dom';
+
+const useHandleSubmit = (params: RequestParamsTypeCreateRecipe) => {
```

Comment:
Why turn this into a hook rather than just have this in the body of the component?